hack/install-golangci-lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# do proxy setting in China mainland
+if [[ -n ${CHINA_MAINLAND:-} ]]; then
+  export GOPROXY=https://proxy.golang.com.cn,direct # set domestic go proxy
+fi
```

Comment:
In China, the developers usually configure this environment variable, and it should not be configured by the script, after all, maybe the user has their own defined `GOPROXY` that cannot override the user environment's proxy.
What do you think?

Reply:
yeah！ Your point is right. I have revoked this PR
Just it's easy to ignore proxy settings when installing in a new environment.